src/xcode/ENA/ENA/Source/Client/HTTP Client/SAPDownloadedPackage.swift

```diff
@@ -34,48 +35,6 @@ struct SAPDownloadedPackage: Fingerprinting {
 	let signature: Data
 	/// The SHA256 string of the package `bin`
 	let fingerprint: String
-
-	// MARK: - Verification
-
-	typealias Verification = (SAPDownloadedPackage) -> Bool
-
-	struct Verifier {
-		private let getPublicKey: PublicKeyProviding
-
-		init(key provider: @escaping PublicKeyProviding = DefaultPublicKeyProvider) {
-			getPublicKey = provider
-		}
-
-		func verify(_ package: SAPDownloadedPackage) -> Bool {
-			guard
-				let parsedSignatureFile = try? SAP_External_Exposurenotification_TEKSignatureList(serializedData: package.signature)
-			else {
-				return false
-			}
-			
-			let publicKey = getPublicKey()
-			
-			for signatureEntry in parsedSignatureFile.signatures {
-				let signatureData: Data = signatureEntry.signature
-				guard
-					let signature = try? ECDSASignature(derRepresentation: signatureData)
-				else {
-					Log.warning("Could not validate signature of downloaded package", log: .api)
-					continue
-				}
-				
-				if publicKey.isValid(signature: signature, for: package.bin) {
-					return true
-				}
-			}
-			
-			return false
-		}
-
-		func callAsFunction(_ package: SAPDownloadedPackage) -> Bool {
-			verify(package)
-		}
-	}
```

Comment:
Totally redundant and never used…

Reply:
Seems you found the code I missed to remove 😅